Pod/Classes/VOKCoreDataManager.m

```diff
@@ -112,13 +127,23 @@ - (NSPersistentStoreCoordinator *)persistentStoreCoordinator
     return _persistentStoreCoordinator;
 }
 
+- (NSBundle *)bundleForModel
+{
+    if (!_bundleForModel) {
+        //Default to using the bundle for this class.
+        _bundleForModel = [NSBundle bundleForClass:[self class]];
+    }
+    
+    return _bundleForModel;
```

Comment:
How is what you're suggesting going to make that easier to tell? 

I suspect it may not work quite right if we set it up to use the main bundle - it's also much more backwards compatible with what we've already got to leave it as is.